### PR TITLE
data insertion in employee table

### PR DESCRIPTION
**Pull request by normanmukama**
i have worked on data insertion of the employee. ie bank details, personal details and company details. 
Note: Have not yet done validation of it. i will do it later.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -47,3 +47,5 @@ Options -Indexes
 # Disable server signature start
 	ServerSignature Off
 # Disable server signature end
+
+
```
Comment (aaron-oluk) [inline .htaccess:51]: disabling server signature

File: Documentation.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+- `/department`: maps to the `Department()` action of the `Home` controller, which displays the department management page.
+- `/branch-master`: maps to the `BranchMaster()` action of the `Home` controller, which displays the branch management page.
+
+## Controller
+
+The routes work together with the `Home` controller to define the actions for the various pages of the HR-MS web application. The `Home` controller defines the actions for the home page, employee management page, create employee page, department management page, and branch management page. When a user visits a URL defined in the routes, the corresponding action of the `Home` controller is executed, which returns the corresponding view.
+
+## Views
+
+The views are defined in the `Views` directory of the HR-MS web application. The views correspond to the actions of the `Home` controller and define the HTML code for the various pages of the application.
```
Comment (aaron-oluk) [inline Documentation.md:291]: documentation for HRMS 1

File: Documentation2.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Dependencies
+
+The HR-MS web application depends on the following libraries and frameworks:
+
+- PHP: the server-side scripting language.
+- MySQL: the database management system.
+- Bootstrap: the front-end framework.
+- Boxicons: the icon library.
+- Composer: the package manager for PHP.
```
Comment (aaron-oluk) [inline Documentation2.md:55]: second documentation of HR-MS

File: app/Config/Routes.php [hunk 4]
```diff
@@ -55,38 +55,82 @@
...
 
 // Working with payroll
 $routes->post('fetchmanagepayroll', 'ManagepayrollController::fetchmanagepayroll');
 
+//insert data in employee table
+$routes->post('insertintoemployee', 'EmployeeDataInsert::index');
+
 
 
 /*
```
Comment (aaron-oluk) [inline app/Config/Routes.php:136]: system routes

File: app/Controllers/BranchmasterController.php [hunk 5]
```diff
@@ -99,4 +96,77 @@ public function fetchbranchmanager()
...
+            'id' => $this->request->getPost('branchid'),
+        ];
+        $this->branchmanager->delete($data);
+        $data = [
+            'status' => 'success',
+            'message' => 'Branch Manager deleted successfully'
+        ];
+        return $this->response->setJSON($data);
+    }
+
```
Comment (aaron-oluk) [inline app/Controllers/BranchmasterController.php:171]: logic and controllers for branch master

File: app/Controllers/ContractController.php [hunk 6]
```diff
@@ -69,23 +69,18 @@ public function fetchcontract()
...
+                                <button type="button" id="contractupdatebtn" class="dropdown-item" value="' . $row['contract_id'] . '">
+                                    <i class="bi bi-pencil" style="color:green"></i> Edit
+                                </button>
+                            </li>
+                            <li>
+                                <button type="button" id="contractdeletebtn" class="dropdown-item" value="' . $row['contract_id'] . '">
+                                    <i class="bi bi-trash" style="color:red"></i> Delete
+                                </button>
+                            </li>
+                        </ul>
```
Comment (Leon12-aaryson) [inline app/Controllers/ContractController.php:83]: contract

**Review by aaron-oluk — APPROVED** [review]
conflicts resolved

**Pull request merged**